src/data/users.js

```diff
@@ -78,6 +78,44 @@ const addRole = async (id, role) => {
     { returnDocument: "after" }
   );
 };
+
 const removeRole = async (id, role) => {};
 
-export { createUser, updateUser, addRole, findUser, findUserByEmailOrUserName };
+const getAllExperts = async () => {
+  try {
+    const db = await users();
+    const experts = await db.find({ role: 'expert' }).toArray();
+    return experts;
+  } catch (e) {
+    throw new DataBaseException(e);
+  }
+};
+
+const getExpertById = async (userId) => {
+  try {
```

Comment:
add validation function to validate userId, HINT: nullcheck first then use Validator.validateId()